mne/inverse_sparse/mxne_optim.py

```diff
@@ -173,6 +173,50 @@ def prox_l1(Y, alpha, n_orient):
     return Y, active_set
 
 
+def primal_l21(M, G, X, active_set, alpha, n_orient, return_primal_only=True):
```

Comment:
```suggestion
def _primal_l21(M, G, X, active_set, alpha, n_orient, return_primal_only=True):
```

I would make this aux function private